documentation/asciidoc/computers/camera/synchronous_cameras.adoc

```diff
@@ -0,0 +1,83 @@
+== Synchronous Captures
+=== IMX477
```

Comment:
https://www.raspberrypi.com/documentation/accessories/camera.html#hardware-specification tells me that IMX477 is the "HQ Camera", so that's probably a more suitable header than "IMX477" :slightly_smiling_face:

Reply:
> https://www.raspberrypi.com/documentation/accessories/camera.html#hardware-specification tells me that IMX477 is the "HQ Camera", so that's probably a more suitable header than "IMX477" 🙂

Must admit the only part I can find is the [NoIR camera](https://forum.fritzing.org/uploads/short-url/n4LCYnOthQ1Bzfh3R4kT4LCP2wr.fzpz). A labelled wire heading off to the side is fine.